example_code/basic_examples/abi_decode/src/lib.rs

```diff
@@ -0,0 +1,59 @@
+
+#![cfg_attr(not(any(feature = "export-abi", test)), no_main)]
+extern crate alloc;
+
+
+/// Import items from the SDK. The prelude contains common traits and macros.
+use stylus_sdk::{alloy_primitives::{U256, Address}, prelude::*};
+// Becauce the naming of alloy_primitives and alloy_sol_types is the same, so we need to re-name the types in alloy_sol_types
+use alloy_sol_types::{sol_data::{Address as SOLAddress, *}, SolType};
+use alloy_sol_types::sol;
```

Comment:
nit but could combine these into a single use statement since the pull from the same dependency